src/Game.js

```diff
@@ -0,0 +1,103 @@
+'use strict'
+
+function Game(){
+  this._frames = [];
+  this._rolls = [];
+};
+
+var game;
+game = new Game();
+
+Game.prototype.updateRolls = function (rollsList) {
+  if(rollsList.length > 21){
+    throw new Error('cannot have more than 21 rolls in a game')
+  }
+  game._rolls = rollsList;
+};
+
+Game.prototype.score = function () {
```

Comment:
I'd like to see this function broken down into its parts, and wrapped in functions to be called (and unit- tested)